frontend/src/App.test.js

```diff
@@ -1,8 +1,25 @@
-import { render, screen } from '@testing-library/react';
+import {fireEvent, render, screen} from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+test('should render title', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  const title = screen.getByRole('heading',{name:'Click-n-Split'});
+  expect(title).toBeInTheDocument();
 });
+
+test('should render homepage successfully', () => {
+  render(<App />);
+  const question = screen.getByText('Enter number of people involved in the transaction?')
+  expect(question).toBeInTheDocument();
+  const inputBox = screen.getByRole('spinbutton',{name:'Enter number of people involved in the transaction?'})
```

Comment:
I would suggest to add test for scrollbar when you enter bigger number. Apart from that everything looks good to me.